resources/views/timelines/_element.blade.php

```diff
@@ -15,8 +15,9 @@
                     <i class="fa-solid fa-chevron-up" id="timeline-element-body-toggle-{{ $element->id }}-show" @if($element->collapsed()) style="display: none;" @endif></i>
                     <i class="fa-solid fa-chevron-down" id="timeline-element-body-toggle-{{ $element->id }}-hide" @if(!$element->collapsed()) style="display: none;" @endif></i>
                     {!! $element->htmlName() !!}
-                    @if(isset($element->date))<span class="text-muted">{{ $element->date }}</span>@endif
-
+                    @if (isset($element->date) || $element->use_event_date && isset($element->entity->event->date))
+                        <span class="text-muted">{{isset($element->entity->event->date) ? $element->entity->event->date : $element->date}}</span>
```

Comment:
It's showing the event's date when the checkbox isn't selected 
![image](https://user-images.githubusercontent.com/3263440/206295574-06d63bf7-07b4-4757-9dcd-cf3e9ca97182.png)
![image](https://user-images.githubusercontent.com/3263440/206295626-e67581e5-dc32-42b2-afc0-4a0cb51a3895.png)